### PR TITLE
8298217: Regressions 30-110% in SwingMark on MacOS, more so on aarch64

### PR DESCRIPTION
**Pull request by aghaisas**
The nature of data plays an important part in rendering performance as noted in the comment made on [JDK-8288948](https://bugs.openjdk.org/browse/JDK-8288948) -
https://bugs.openjdk.org/browse/JDK-8288948?focusedCommentId=14504772&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14504772

Although the fix [JDK-8288948](https://bugs.openjdk.org/browse/JDK-8288948) is logical, it seems to have an adverse impact on SwingMark test numbers especially on aarch64.

The SwingMark tests closely depict the UI operations in real-world applications as compared to the J2DBench and RenderPerfTest tests. Hence, it is decided to revert the regression causing commits [JDK-8288948](https://bugs.openjdk.org/browse/JDK-8288948) (and dependent [JDK-8291266](https://bugs.openjdk.org/browse/JDK-8291266)).

Metal rendering pipeline performance numbers before and after this PR are attached to the JBS.

Note : The fix for [JDK-8291266](https://bugs.openjdk.org/browse/JDK-8291266) has a good regression test - DrawPrimitivesTest.java. It passes even after reverting the said changes. This test will be added back once this PR is committed.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8298217](https://bugs.openjdk.org/browse/JDK-8298217): Regressions 30-110% in SwingMark on MacOS, more so on aarch64

### Reviewers
 * [Alexey Ushakov](https://openjdk.org/census#avu) (@avu - Committer) ⚠️ Review applies to [da8c8f50](https://git.openjdk.org/jdk20/pull/47/files/da8c8f5076a03436dcf733163e5617803f1bc21d)
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**) ⚠️ Review applies to [da8c8f50](https://git.openjdk.org/jdk20/pull/47/files/da8c8f5076a03436dcf733163e5617803f1bc21d)
 * [Jayathirth D V](https://openjdk.org/census#jdv) (@jayathirthrao - **Reviewer**) ⚠️ Review applies to [da8c8f50](https://git.openjdk.org/jdk20/pull/47/files/da8c8f5076a03436dcf733163e5617803f1bc21d)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk20 pull/47/head:pull/47` \
`$ git checkout pull/47`

Update a local copy of the PR: \
`$ git checkout pull/47` \
`$ git pull https://git.openjdk.org/jdk20 pull/47/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 47`

View PR using the GUI difftool: \
`$ git pr show -t 47`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk20/pull/47.diff">https://git.openjdk.org/jdk20/pull/47.diff</a>

</details>

### CODE REVIEW

File: src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLRenderQueue.m [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019, 2021, Oracle and/or its affiliates. All rights reserved.
```
Comment (avu) [inline src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLRenderQueue.m]: I suppose we should leave 2022 here.

**Review by avu — APPROVED** [review]
The rest of the code looks good

**Review by prrace — APPROVED** [review]

**Review by jayathirthrao — APPROVED** [review]
We should keep latest updated year. Please update it before pushing.

**Pull request closed without merging**